web_tool_script_2.R

```diff
@@ -1,34 +1,22 @@
+options(r2dii_config = paste0(par_file_path,"/AnalysisParameters.yml"))
+
+devtools::load_all()
+use_r_packages()
```

Comment:
I reorganized scripts headers to start with options, then sourcing all functions and data, then attaching packages, then anything else.